src/main/java/arb/model/ModelManager.java

```diff
@@ -107,6 +107,11 @@ public void deleteClient(Client target) {
         addressBook.removeClient(target);
     }
 
+    @Override
+    public void markProject(Project project) {
+        addressBook.markProject(project);
```

Comment:
I think you need to add one additional line for the project list in the UI to update. Try adding `updateFilteredProjectList(PREDICATE_SHOW_ALL_PROJECTS);` and see if that works?

Reply:
It doesn't oops :(